overflow/overflow-lesson-plans.md

```diff
@@ -0,0 +1,125 @@
+- [Git & GitHub](https://github.com/C4Q/ac-curriculum-web/blob/master/unit1-intro-to-programming-in-javascript/git.md): 2 days
```

Comment:
Just a temporary folder to hold the stuff that was previously in weeks 1 +
2 that we're now moving out. Will remove this folder once I'm done
reordering everything

On Tue, Jun 28, 2016 at 5:58 PM, Benjamin Sussman <notifications@github.com>
wrote:

> In overflow/overflow-lesson-plans.md
> <https://github.com/C4Q/ac-curriculum-web/pull/14#discussion_r68851906>:
>
> > @@ -0,0 +1,125 @@
> > +- [Git & GitHub](https://github.com/C4Q/ac-curriculum-web/blob/master/unit1-intro-to-programming-in-javascript/git.md): 2 days
>
> What is this file?? This is weird...
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/C4Q/ac-curriculum-web/pull/14/files/ba8677ef4df6f0410feb371b264fff35e63cb86c#r68851906>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAQnnuZX-h5-EzpXthBrHUPSZCkWkCgHks5qQZkLgaJpZM4JAYSc>
> .
>